.github/workflows/cmake.yml

```
@@ -862,7 +858,7 @@ jobs:
         MSAN_OPTIONS: ${{ matrix.msan-options || 'verbosity=0' }}:abort_on_error=1:halt_on_error=1
         TSAN_OPTIONS: ${{ matrix.tsan-options || 'verbosity=0' }}:abort_on_error=1:halt_on_error=1
         LSAN_OPTIONS: ${{ matrix.lsan-options || 'verbosity=0' }}:abort_on_error=1:halt_on_error=1
-        UBSAN_OPTIONS: ${{ matrix.ubsan-options || 'verbosity=0' }}:print_stacktrace=1:abort_on_error=1:halt_on_error=1
+        UBSAN_OPTIONS: ${{ matrix.ubsan-options || 'verbosity=0' }}:abort_on_error=1:halt_on_error=1:print_stacktrace=1:silence_unsigned_overflow=1
```

actionlint (1.7.11)

[error] property "ubsan-options" is not defined in object type {asan-options: string; build-config: string; build-shared: string; cflags: string; chost: string; cmake-args: string; codecov: string; compiler: string; cxx-compiler: string; cxxflags: string; gcov-exec: string; ldflags: string; msan-options: string; name: string; os: string; packages: string; parallels-jobs: number; toolchain: string}

(expression)